CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
I would suggest you to fix commit ref format in the commit description. 
It can be generated by using command:
```
git show -s --pretty=reference e528793c3
```